### PR TITLE
Add native segwit prefix for litecoin

### PR DESCRIPTION
**Pull request by Xekyo**

### CODE REVIEW

**Comment by Xekyo** [comment]
It would probably be sensible to add litecoin testnet at the same time.

**Comment by OttoAllmendinger** [comment]
indeed

**Comment by argjv** [comment]
I found the reason why the tests are failing. It's a legacy "feature" of the original library. It has nothing to do with the LTC params.

Basically, the problem is the method `fromBase58()` can take an array of network objects, but when it does that, it just pops the last one, in this case, the `litecointest` one.

We should move these 2 network objects above the `bitcoin` one... I know... It's super bad. So please, also add a `TODO` in the faulty method to remove array support and create a GitHub issue.

File: src/networks.js [hunk 1]
```diff
@@ -152,6 +152,7 @@ module.exports = {
   },
   litecoin: {
```
Comment (argjv) [inline src/networks.js]: Move `litecoin` and `litecoinTest` above `bitcoin`

**Pull request merged**